docs/getting-started/installing-shoko-server.mdx

```diff
@@ -126,5 +157,19 @@ Open Docker from the applications menu, then go to Containers. You'll need to do
 
 Please note that if you want to edit volumes or environment variables, you'll need to stop the container first.
 
-### Next Step
+## Docker (TrueNAS)
+
+### TrueNAS Scale
+
+TBD
+
+### TrueNAS Core
+
+Docker is not supported on this platform, as **TrueNAS Core** uses FreeBSD as base OS.
```

Comment:
Maybe we should mention that you can still imstall docker in a jail to then install shoko in docker inside the jail?

Reply:
whoever wants to support this kinda setup can add it, lol - I can't because idk about jails